lib/elixir/lib/string.ex

```diff
@@ -1478,6 +1481,9 @@ defmodule String do
       iex> String.at("elixir", -10)
       nil
 
+      iex> String.at("\u0065\u0301lixir", 0)
```

Comment:
Can we please use an actual letter or another alphabet to show this rather than \uxxxx?